io.github.wdaniau.fstl.yml

```diff
@@ -0,0 +1,25 @@
+id: io.github.wdaniau.fstl
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+command: fstl-e
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
+  - --filesystem=xdg-run/gvfs # make GnomeVFS accessible
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --device=all
+modules:
+  - name: fstl-e
+    buildsystem: cmake
+    sources:
+      - type: git
+        branch: fstl-e
```

Comment:
NEVER a git branch

Reply:
Well that's more a problem as fstl-e is my default branch on github but flatpak-builder use master ...

Reply:
No, that's not a problem. A commit or a tag is REQUIRED.

The problem is setting a branch

Reply:
ok thanks